ROR-Assignment-Day-04/main.rb

```diff
@@ -0,0 +1,123 @@
+require_relative 'loan_eligibility'
+require_relative 'economic_analysis'
+require_relative 'military_defense'
+
+class LoanEligibilityTool
+  include DebtGdpRatio
+  include LoanEligibility
+end
+
+class EconomicAnalysisTool
+  include DebtGdpRatio
+  include EconomicAnalysis
+end
+
+class MilitaryAndDefenseAnalysis
+  include MilitaryAndDefense
+end
+
+class MilitaryAndDefenseAnalysis
+    include MilitaryAndDefense
+end
+
+
+    puts "Analysis Loan Tool"
+
+    puts "Choose the tool you want to use: "
+    print "1. Economic Analysis Tool\n2. Military and Defense Analysis Tool\n3. Loan Eligibility Tool\n"
+    choice = gets.chomp.to_i
+
+    case choice
+    when 1
+        begin
+            puts "Economic Analysis Tool"
+          
+            print "Enter the country name: "
+            country_name = gets.chomp.split.map(&:capitalize).join(' ')
```

Comment:
```suggestion
            country_name = gets.chomp.squish.titleize
```
Can you please check if this does the same?

Reply:
Yes it does the same work. I have actually checked capitalize but it didn't work hence I used it